src/main/resources/db/migration/V1__.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE booking
+(
+    id              BIGINT NOT NULL,
+    description     VARCHAR(255),
```

Comment:
This was generated automatically, apparently as a default for this field. 255 might be a bit short imo, what do you think?

Reply:
We dont need database migration or any of that sort... Spring automatically generates our sql and respective tables... I just forgot to  include it in my last PR. Please see #32.

Reply:
That automatic feature doesn't cover changes to our database structure, which are quite likely to keep happening. I'd rather set up proper migrations now than potentially face data loss due to unchecked magic auto-migrations in the future. (Key word is unchecked, flyway still allows us to generate these, we just get the ability to check that they are actually what we wanted)

Reply:
Sounds good!